codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/FluentClientDecorator.kt

```diff
@@ -522,3 +562,21 @@ class FluentClientGenerator(
         }
     }
 }
+
+fun generateShapeMemberDocs(writer: RustWriter, symbolProvider: SymbolProvider, shape: StructureShape, model: Model): List<String> {
+    val structName = symbolProvider.toSymbol(shape).rustType().qualifiedName()
+    return shape.members().map { memberShape ->
+        val name = symbolProvider.toMemberName(memberShape)
+        val snakeCaseName = name.toSnakeCase()
```

Comment:
Is the `toSnakeCase` actually needed? It seems like the symbol provider should be returning it that way already for structure members.

Reply:
you were right, I removed it